src/types.rs

```diff
@@ -114,12 +116,13 @@ impl FeeRate {
 
     /// Calculate absolute fee in Satoshis using size in weight units.
     pub fn fee_wu(&self, wu: usize) -> u64 {
-        self.fee_vb(wu.vbytes())
+        let vbytes = wu as f32 / 4.0;
+        (self.0 * vbytes).ceil() as u64
```

Comment:
With your change to the wu -> vb conversion, `self.fee_vb(wu.vbytes())` does exactly what these lines do?

Reply:
It's better to be explicit with simple calculations.